library-templates/generated-connector-template/files/.gitignore

```diff
@@ -0,0 +1,59 @@
+# Compiled class file
+*.class
+*.balx
+# Log file
+*.log
+
+generated
+# Ballerina Config file
+Config.toml
+
+# BlueJ files
+*.ctxt
+
+# Mobile Tools for Java (J2ME)
+.mtj.tmp/
+
+# .DS_Store files
+*.DS_Store
+
+# Package Files
+*.jar
+!gradle/wrapper/gradle-wrapper.jar
+*.war
+*.ear
+*.zip
+*.tar.gz
+*.rar
+*.deb
+
+# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
+hs_err_pid*
+
+# Ignore everything in this directory
+target
+.classpath
+.settings
+.project
+*.iml
+*.iws
+*.ipr
+.idea
+.m2
+.vscode/
+# Ignore ballerina files
```

Comment:
```suggestion

# Ignore ballerina files
**/Config.toml
```

Reply:
Updated in https://github.com/ballerina-platform/ballerina-library/pull/6759/commits/c34e0cc3b808ede0a3bea44178b8589912ff4b08